Source/Core/Settings/SettingsMenu.cs

```diff
@@ -114,6 +117,11 @@ public override void DoSettingsWindowContents(Rect inRect)
             if (listing_options.ButtonTextLabeled("PrisonLabor_ButtonRemoveModFromSaveDesc".Translate(), "PrisonLabor_ButtonRemoveModFromSave".Translate()))
                 Find.WindowStack.Add(new SelectSaveForCleaningDialog());
 
+            listing_options.Gap();
+            listing_options.Gap();
+            listing_options.Label("Max number of prisoners per gaurd to start lossing motivation (become ineffective)", -1f);
```

Comment:
Move string to keys and use with `.Translate()` method

Reply:
Ok, when i'm done with AI.